fee-monitor/src/DynamicChecker.ts

```diff
@@ -89,7 +89,7 @@ export class DynamicChecker {
         commitedVotesPerValidator: new Map(),
         assembledVotesPerAuthor: new Map(),
     };
-    public nominationDeposits: Map<PlatformAddress, U64> = new Map();
+    public nominationDeposits: Map<string, U64> = new Map();
```

Comment:
`Object.is` is used to compare keys of `Map`, so this is the right change. But you don't have to remove all `PlatformAddress` usages.